code/_onclick/hud/rendering/plane_master.dm

```diff
@@ -266,3 +250,4 @@
 	plane = FIELD_OF_VISION_BLOCKER_PLANE
 	render_target = FIELD_OF_VISION_BLOCKER_RENDER_TARGET
 	mouse_opacity = MOUSE_OPACITY_TRANSPARENT
+	render_relay_plane = null
```

Comment:
The render_relay_plane used to be RENDER_PLANE_MASTER by inheritance. What does setting this to null change?

Reply:
It does not relay